src/components/header.tsx

```diff
@@ -1,47 +1,57 @@
 import { Link } from 'gatsby';
 import React from 'react';
-import { css, SerializedStyles } from '@emotion/core';
+import logoLight from '../images/logos/nodejs-logo-light-mode.svg';
+import logoDark from '../images/logos/nodejs-logo-dark-mode.svg';
 
-/* eslint-disable-next-line @typescript-eslint/no-var-requires */
-const logo = require('../images/logo.svg');
+const activeStyleTab = {
+  fontWeight: 'var(--font-weight-semibold)',
+  color: 'var(--color-text-accent)',
+  borderBottom: 'var(--space-04) inset var(--color-text-accent)',
+};
 
-const ulStyles: SerializedStyles = css`
-  @media (max-width: 380px) {
-    padding: 0;
-  }
-  margin: 0 auto;
-  padding: 0 4.8rem;
-  display: flex;
-  align-items: center;
-  list-style: none;
-`;
-
-const Header = (): JSX.Element => (
+const Header = () => (
   <nav className="nav">
-    <ul css={ulStyles}>
-      <li>
-        <Link to="/" style={{ display: 'block' }}>
-          <img src={logo} alt="Node.js" className="nav__logo" />
-        </Link>
-      </li>
-      <li className="nav__tabs">
-        <Link to="/learn">Learn</Link>
-      </li>
+    <div className="logo">
+      <Link to="/">
+        <img
+          src={logoLight}
+          alt="Node.js"
+          className="nav__logo light-mode-only"
+        />
+        <img
+          src={logoDark}
+          alt="Node.js"
+          className="nav__logo dark-mode-only"
+        />
+      </Link>
+    </div>
+
+    <ul className="nav__tabs__container">
       <li className="nav__tabs">
-        <Link to="/docs">Docs</Link>
+        <Link to="/learn" activeStyle={activeStyleTab} partiallyActive>
+          Learn
+        </Link>
       </li>
       <li className="nav__tabs">
-        <Link to="/community">Community</Link>
+        <Link to="/docs" activeStyle={activeStyleTab} partiallyActive>
+          Documentation
+        </Link>
       </li>
       <li className="nav__tabs">
-        <Link to="/download">Download</Link>
+        <Link to="/download" activeStyle={activeStyleTab} partiallyActive>
+          Download
+        </Link>
       </li>
-      <li style={{ flexGrow: 1 }} />
+    </ul>
+
+    <div style={{ flexGrow: 1 }}></div>
+
+    <ul className="right-container">
       <li className="nav__tabs nav__tabs--right">
         <button
           type="button"
           className="dark-mode-toggle"
-          onClick={(): boolean => document.body.classList.toggle('dark-mode')}
+          onClick={() => document.body.classList.toggle('dark-mode')}
```

Comment:
Ouch.... dirty trick

Reply:
Thank you @alexandrtovmach ! Totally agree with you. My coding skills only take me so far 😬 But as Adam said, I think we can totally iterate and computerize the elements. 👍